assets/components/paymentMethods/paymentMethods.jsx

```diff
@@ -6,16 +6,23 @@ import React from 'react';
 
 import StripePopUpButton from 'components/stripePopUpButton/stripePopUpButton';
 import PayPalExpressButton from 'components/payPalExpressButton/payPalExpressButton';
+import PayPalContributionButton from 'components/payPalContributionButton/payPalContributionButton';
 import ErrorMessage from 'components/errorMessage/errorMessage';
 
 
 // ----- Types ----- //
 
+export type PayPalButtonType =
+  'ExpressCheckout' |
+  'ContributionsCheckout' |
+  'NotSet';
```

Comment:
The `NotSet` logic should go away when we turn on PayPal shortly. I don't think it is worth it to change that name.